vue/mixins/upload.js

```diff
@@ -26,7 +30,7 @@ export const uploadMixin = {
             this.fileInputRef = filesInputRef;
         },
         setFiles(filesList) {
-            this.filesData = [...filesList];
+            this.filesData = this.multiple ? [...filesList] : [filesList[0]];

```

Comment:
Is this required? If you set `multiple="false"` in the `input` won't that guarantee you'll have a single one?

Reply:
This is to fix multiple files being set when dragging even when `multiple` is set to false.
This was an old bug. It was also happening in `upload-area` before the split component refactor.